src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -237,6 +245,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         g.command('logs', 'acr_task_logs', client_factory=cf_acr_runs,
                   table_transformer=None)
 
+    with self.command_group('acr taskrun', acr_taskrun_util) as g:
```

Comment:
Can you add `is_preview=True`? It's better to flag it as preview feature in cli as the service is in preview.